TTHAnalysis/python/tools/higgsRecoTTH.py

```diff
@@ -103,7 +118,18 @@ def run(self,event,Collection):
                     elif abs(genpar[genpar[part.genPartIdxMother].genPartIdxMother].pdgId) == 6:
                         if self.debug: print "the mother of this W is a Top"
                         LFromWFromT.append(part)
+            elif abs(part.pdgId) in [12, 14, 16, 18] and part.statusFlags &(1 << statusFlagsMap['isPrompt']):
+                if self.debug: print "it is a neutrino"
+                if part.genPartIdxMother >= 0 and abs(genpar[part.genPartIdxMother].pdgId) == 24: 
+                    if self.debug: print "the mother of this neutrino is W+ or W-"
+                    if abs(genpar[genpar[part.genPartIdxMother].genPartIdxMother].pdgId) == 25:
+                        if self.debug: print "the mother of this W is a Higgs"
+                        NuFromWFromH.append(part)
+                       
```

Comment:
remove  empty line addition and printouts commented lines (if you want you can uncomment them and put them under "if debug:"

Reply:
DONE